all.json

```
@@ -32,5 +32,8 @@
     "polkawallets.site",
     "polkodot.network",
     "walletsynchronization.com"
+    "polkadot-distribution.live"
```

```suggestion
    "polkadot-distribution.live",
```

Also needs a `,` at the end of the previously line.

---

new to this, have to learn the details. i changed it and updated

---

No issues at all.